src/test/java/htsjdk/TestClassDependenceTest.java

```diff
@@ -0,0 +1,63 @@
+package htsjdk;
+
+import htsjdk.utils.ClassFinder;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.NoInjection;
+import org.testng.annotations.Test;
+import org.testng.collections.Sets;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.*;
+
+/**
+ * Created by farjoun on 10/2/17.
+ */
+public class TestClassDependenceTest extends HtsjdkTest {
```

Comment:
@yfarjoun Sorry to keep nitpicking this branch, but this class really needs a comment explaining what it's testing.